modules/ROOT/pages/design-create-publish-api-specs.adoc

```diff
@@ -9,7 +9,7 @@ API Designer provides a web-based interface for designing, documenting, testing,
 
 == Designing
 
-You can use two different tools in API Designer -- the text editor and the visual editor -- to create API specifications in RESTful API Modeling Language (RAML) 0.8 or 1.0, or according to OpenAPI specification 2.0 (OAS).
+You can use two different tools in API Designer -- the text editor and the visual editor -- to create API specifications in RESTful API Modeling Language (RAML) version 0.8 or 1.0, or according to OpenAPI specification (OAS) version 2.0 or 3.0.
```

Comment:
Try to avoid using parenthetical language, which can confuse readers (particularly those for whom English is not native).
In any case, an em-dash is not rendered in AsciiDoc, and space-hyphen-hyphen-space is pretty awkward. 
Most style references advise against the spaces on either side, despite the fact that that can cause occassional line-break issues.
My suggestion is always to reword to avoid:
```suggestion
You can use either the text editor or the visual editor to create API specifications in RESTful API Modeling Language (RAML) version 0.8 or 1.0, or according to OpenAPI specification (OAS) version 2.0 or 3.0.
```